web-ui/src/main/resources/catalog/js/admin/SourcesController.js

```diff
@@ -211,7 +211,7 @@
 
       $scope.deleteSourceLogo = function () {
         $scope.source.logo = null;
-        // $scope.updateSource();
+        $scope.gnSourceForm.$setDirty();
```

Comment:
It seems when removed the logo, there is no need to save the source so it gets removed. Not sure if this is required?

Reply:
Once user unset the logo, user need to save the form. So set dirty enable the save button. 

I also added a check on the list to avoid broken image

![image](https://github.com/user-attachments/assets/69251919-101b-4975-a93a-c658fe8cb24b)